index.html

```diff
@@ -0,0 +1,219 @@
+<!DOCTYPE html>
+<html lang="en">
+
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+
+		<title>Anna Russett</title>
+	
+		<link rel="icon" type="image/x-icon" href="http://www. /favicon.ico">
+		<meta name="description" content="content">
+
+		<!-- for Google+ -->
+		<meta itemprop="name" content="Anna Russett">
+		<meta itemprop="description" content="content">
+		<meta itemprop="image" content="images/ ">
+		<!-- for Twitter -->
+		<meta name="twitter:card" content="summary_large_image">
+		<meta name="twitter:site" content="@USERNAME">
+		<meta name="twitter:title" content="Anna Russett">
+		<meta name="twitter:description" content="content">
+		<meta name="twitter:creator" content="@TITLE">
+		<!-- Twitter summary card with large image must be at least 280x150px -->
+		<meta name="twitter:image:src" content="images/ ">
+		<!-- for Facebook -->
+		<meta property="og:title" content="Anna Russett">
+		<meta property="og:type" content="article">
+		<meta property="og:url" content="http:// ">
+		<meta property="og:image" content=" ">
+		<meta property="og:description" content="content">
+		<meta property="og:site_name" content="Anna Russett">
+
+		<link rel="stylesheet" href="css/styles.css">
+
+
+	</head>
+
+	<body>
+
+
+		<div id="mobilemenuexpand">
```

Comment:
if i open the menu while it's in mobile mode ( ie. when it's got a small enough width ) && then i stretch the window out to desktop size, the text from the mobile menu remains present, this is because the div#mobilemenuexpand is still display:flex; u should probably make a media-query in ur style sheet that changes the display to none when the page is desktop sized